theora_image_transport/src/theora_subscriber.cpp

```diff
@@ -245,4 +291,70 @@ void TheoraSubscriber::internalCallback(const theora_image_transport::msg::Packe
   callback(latest_image_);
 }
 
+void TheoraSubscriber::declareParameter(const std::string &base_name,
+                                       const ParameterDefinition &definition)
+{
+  //transport scoped parameter (e.g. image_raw.compressed.format)
```

Comment:
```suggestion
  //transport scoped parameter (e.g. image_raw.theora.quality)
```

Reply:
Here are made it `post_processing_level` as this is the only theora subscriber parameter